ddapm_test_agent/agent.py

```diff
@@ -1227,6 +1245,7 @@ def make_app(
     app["disable_error_responses"] = disable_error_responses
     app["snapshot_removed_attrs"] = snapshot_removed_attrs
     app["snapshot_regex_placeholders"] = snapshot_regex_placeholders
+    app["snapshot_server_cassettes_directory"] = snapshot_server_cassettes_directory
```

Comment:
heheh yeah the name conflict is apparent here... Maybe let's just call it "recorder" or maybe reuse the name "vcr"?